features/step_definitions/run_steps.rb

```diff
@@ -25,13 +25,13 @@
 
 
 
-Then(/^I get the error "(.+?)"$/) do |str|
-  verify_error str
+Then(/^I get the error$/) do |error_message|
+  verify_error error_message.strip
 end
 
 
-Then(/^I get the error$/) do |str|
-  verify_error str.strip
+Then(/^I get the error "(.+?)"$/) do |error_message|
```

Comment:
Implementing this has allowed me to inline the `verify_error` helper, and remove the `error_helper.rb` file.